mm2src/coins/qrc20/qrc20_tests.rs

```diff
@@ -1061,3 +1061,16 @@ fn test_send_contract_calls_recoverable_tx() {
         discriminant(&TransactionErr::TxRecoverable(TransactionEnum::from(tx), String::new()))
     );
 }
+
+#[test]
+fn test_qrc20_validate_valid_and_invalid_pubkey() {
```

Comment:
Since `Qrc20Coin` uses `utxo_common::validate_other_pubkey`, we can remove this test.

Reply:
done